src/appmixer/pinterest/core/GetPin/component.json

```diff
@@ -0,0 +1,75 @@
+{
+    "name": "appmixer.pinterest.core.GetPin",
+    "author": "Appmixer <info@appmixer.com>",
+    "description": "Retrieves information about a specific pin.",
+    "version": "1.0.0",
+    "private": false,
+    "auth": {
+        "service": "appmixer:pinterest"
+    },
+    "quota": {
+        "manager": "appmixer:pinterest",
+        "resources": "requests"
+    },
+    "inPorts": [
+        {
+            "name": "in",
+            "schema": {
+                "type": "object",
+                "properties": {
+                    "pinId": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "pinId"
+                ]
+            },
+            "inspector": {
+                "inputs": {
+                    "pinId": {
+                        "type": "select",
+                        "tooltip": "The ID of the pin to fetch.",
+                        "label": "Pin",
```

Comment:
label: "Pin Id" 
tooltip: "Please enter the Pin Id or select a Pin from the list to retrieve."
type: "text"  - "select" has better ux, however it doesn't allow entering a value manually